pkg/gce-pd-csi-driver/cache.go

```diff
@@ -18,25 +20,37 @@ const (
 	mainLvSuffix       = "csi-main"
 	raidedLocalSsdName = "csi-driver-data-cache"
 	raidMode           = "0"
-	raidedLssdPrefix   = "/dev/md/"
 )
 
-var raidedLocalSsdPath = raidedLssdPrefix + raidedLocalSsdName
+func fetchRAIDedLocalSsdPath() (string, error) {
+	args := []string{
+		"--detail",
+		"--scan",
+	}
+	info, err := common.RunCommand("grep", []string{raidedLocalSsdName}, "mdadm", args...)
+	if err != nil || len(info) == 0 {
+		return "", fmt.Errorf("Error getting RAIDed device path for Datacache %v, output:%v ===============", err, string(info))
+	}
+	infoString := strings.TrimSpace(string(info))
```

Comment:
Provide a sample info string as part of the a comment will help later someone else comes to read the code.